.github/workflows/build-editor.yaml

```
@@ -0,0 +1,188 @@
+          - runner: windows-latest
+            arch: x64
+            artifact_name: LaMachineEditor-windows-x64
+            processing_asset: processing-1313-4.5.2/processing-4.5.2-windows-x64-portable.zip
+
+    runs-on: ${{ matrix.runner }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Java
+        uses: actions/setup-java@v3
```

actionlint (1.7.10)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)